src/app/zap-templates/zcl/data-model/chip/energy-preference-cluster.xml

```diff
@@ -0,0 +1,54 @@
+<!--
+Copyright (c) 2021 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="Energy"/>
+
+  <bitmap name="Feature" type="bitmap32">
+    <cluster code="0x009b"/>
+    <field name="EnergyBalance" mask="0x1"/>
+    <field name="LowPowerModeSensitivity" mask="0x2"/>
+  </bitmap>
+
+  <enum name="EnergyPriorityEnum" type="enum8">
+    <cluster code="0x009b"/>
+    <item name="Comfort" value="0x0"/>
+    <item name="Speed" value="0x1"/>
+    <item name="Efficiency" value="0x2"/>
+    <item name="WaterConsumption" value="0x3"/>
+  </enum>
+
+  <struct name="EnergyBalanceStruct">
```

Comment:
This does not match the struct name in the spec, as far as I can tell.

If this is targeting some un-merged PR, that needs to be clearly stated.